neurolang/frontend/datalog/tests/test_parser_squall.py

```diff
@@ -550,11 +538,142 @@ def test_server_example_VWFA(datalog_simple):
 
         obtain every Term @t with every Quantity @lor such that @t `specific to the VWFA` with @lor.
     """
-    logic_code = parser(code)
-    datalog_simple.walk(logic_code)
 
-    assert True
+    x = Symbol('x')
+    y = Symbol('y')
+    z = Symbol('z')
+    focus = Symbol('focus')
+    vwfa = Symbol('vwfa')
+    fresh1 = Symbol.fresh()
+    fresh2 = Symbol.fresh()
+    expected1 = Implication(
+        vwfa(x, y, z),
+        Conjunction(
+            (
+                focus(x, y, z),
+                EQ(
+                    fresh1,
+                    ADD(
+                        ADD(
+                            POW(SUB(
+                                x, Constant(-45)), Constant(2)),
+                            POW(SUB(
+                                y, Constant(-57)), Constant(2)),
+                        ),
+                        POW(SUB(
+                            z, Constant(-12)), Constant(2)),
+                    ),
+                ),
+                LT(fresh1, fresh2),
+                EQ(fresh2, POW(Constant(5), Constant(2))),
+            )
+        ),
+    )
+
+    VWFA_imag = Symbol('vwfa imag')
+    created_region = Symbol("created region")
+    x_f = Symbol.fresh()
+    y_f = Symbol.fresh()
+    z_f = Symbol.fresh()
+    fresh3 = Symbol.fresh()
+    fresh4 = Symbol.fresh()
+
+    expected2 = Implication(fresh3(AggregationApplication(created_region, (x_f, y_f, z_f)), ),
+                            vwfa(x_f, y_f, z_f),
+                            )
+    expected3 = Implication(VWFA_imag(fresh4), fresh3(fresh4))
+
+    mentions_vwfa = Symbol('mentions vwfa')
+    study = Symbol('study')
+    s_f = Symbol.fresh()
+    report = Symbol("report")
+
+    expected4 = Implication(
+        mentions_vwfa(s_f), Conjunction((study(s_f), vwfa(
+            x_f, y_f, z_f), report(s_f, x_f, y_f, z_f)))
+    )
+
+    given_studi = Symbol("given studi")
+    count_of_studies = Symbol.fresh()
+    prob = Symbol.fresh()
+
+    expected5 = Implication(
+        count_of_studies(
+            AggregationApplication(Symbol('count'), (s_f,)),
+        ),
+        study(s_f),
+    )
 
+    expected6 = Implication(
+        ProbabilisticChoice(prob, given_studi(s_f)),
+        Conjunction((study(s_f), EQ(prob, DIV(Constant(1), count_of_studies)))
+                    ))
```

Comment:
Good job! This will work if you do
```python
    c = Symbol.fresh()

    expected6 = Implication(
        ProbabilisticChoice(prob, given_studi(s_f)),
        Conjunction((study(s_f), EQ(prob, DIV(Constant(1), c)), count_of_studies(c))
                    ))
```

Reply:
Done.